lib/xs_transport_lwt_unix_client.ml

```diff
@@ -45,7 +45,7 @@ let destroy = Lwt_unix.close
 let read = Lwt_unix.read
 
 let write fd bufs ofs len =
-  Lwt_unix.write_string fd bufs ofs len >>= fun n ->
+  Lwt_unix.write_string fd (Bytes.unsafe_to_string bufs) ofs len >>= fun n ->
```

Comment:
Hmm, actually, could we simply use `Lwt_unix.write` here, which accepts `bytes`?

Reply:
Interestingly, that would be equivalent to undoing the change in https://github.com/xapi-project/ocaml-xenstore-clients/pull/10/files :thinking: